src/components/EditableField/EditableField.module.scss

```diff
@@ -11,30 +11,28 @@
 }
 
 .editInput {
+  width: 100%;
   background: none;
   border: 2px solid $light-grey;
   border-radius: 4px;
-  width: 100%;
   font-size: 1.8rem;
   font-weight: $extrabold;
 }
 
 .editTextarea {
-  background: none;
-  border: 2px solid $light-grey;
-  border-radius: 4px;
-  width: 100%;
+  @extend .editInput;
   min-height: 200px;
```

Comment:
Keep a new line between the extend and the rest. So it is beyond obvious there is something more than just a property